intdataplane/int_dataplane.go

```diff
@@ -750,3 +752,39 @@ type iptablesTable interface {
 	RemoveChains([]*iptables.Chain)
 	RemoveChainByName(name string)
 }
+
+// msgStringer wraps an API message to customise how we stringify it.  For example, it truncates
+// the lists of members in the (potentially very large) IPSetsUpdate messages.
+type msgStringer struct {
+	msg interface{}
+}
```

Comment:
The advantage is that we defer the call to `String()` until after the logging framework has decided to log.